### ISSUE TITLE
regularization loss is large

### ISSUE THREAD

**Issue by tbornt** [Zehaos/MobileNet#6]
I tried your pre-trained mobilenet model. The regularization loss is large about 5.25. Have you noticed this. I'm not quite sure if the large regularization means overfitting. If you trained the model using slim. I believe the weight_decay parameter can help reduce this loss.

**Comment by Zehaos** [comment]
I have not noticed this. Could you tell me how to get the regularization loss?
As it is mentioned in the paper, small model less need regularization, so I set the weight_decay to be zero and use Inception's data augmentation method. This is not fully consistent with the paper, but I think the training result is still acceptable. 
But if someone trys to train a better model, I suggest to tune these hyperparameters.

**Comment by tbornt** [comment]
If you trained with slim, open tensorboard and you can find three losses. losses, regularization_loss_1 and total_loss_1 as shown below.
<img width="652" alt="wx20170505-171858 2x" src="https://cloud.githubusercontent.com/assets/21997233/25740003/22029880-31b7-11e7-9d9b-febd36d1cd7a.png">

**Comment by Zehaos** [comment]
In my tensorboard, I have 'losses'、 ‘total_loss_1’ only...
